commit fe2e727883a371f21e02454ad6b58eed407aa7b9
Author: popcornmix <popcornmix@gmail.com>
Date:   Sun Jun 17 00:14:54 2012 +0100

    Explicitly set usb host channels to 8, as in the spec

Index: linux-3.2.46/drivers/usb/host/dwc_otg/dwc_otg_core_if.h
===================================================================
--- linux-3.2.46.orig/drivers/usb/host/dwc_otg/dwc_otg_core_if.h	2013-07-26 19:31:23.000000000 +0000
+++ linux-3.2.46/drivers/usb/host/dwc_otg/dwc_otg_core_if.h	2013-07-26 19:33:34.000000000 +0000
@@ -267,7 +267,8 @@
 extern int dwc_otg_set_param_host_channels(dwc_otg_core_if_t * core_if,
 					   int32_t val);
 extern int32_t dwc_otg_get_param_host_channels(dwc_otg_core_if_t * core_if);
-#define dwc_param_host_channels_default 12
+//#define dwc_param_host_channels_default 12
+#define dwc_param_host_channels_default 8// Broadcom BCM2708
 
 /** The number of endpoints in addition to EP0 available for device
  * mode operations.
